spring/src/main/java/io/seata/spring/annotation/GlobalTransactionScanner.java

```diff
@@ -317,6 +317,7 @@ public Object postProcessBeforeInitialization(Object bean, String beanName) thro
                 if (null != m) {
                     return m.invoke(dataSourceProxy, args);
                 } else {
+                    method.setAccessible(true);
```

Comment:
Shouldn't we restore the accessible? e.g. 
```
boolean oldAccessible = method.getAccessible();
method.setAccessible(true);
try {
    ...
} finally {
    method.setAccessible(oldAccessible);
}
```

Reply:
Good idea.

Reply:
> Shouldn't we restore the accessible? e.g.
> 
> ```
> boolean oldAccessible = method.getAccessible();
> method.setAccessible(true);
> try {
>     ...
> } finally {
>     method.setAccessible(oldAccessible);
> }
> ```

fixed